src/ml/mlHelper.ts

```diff
@@ -22,3 +27,55 @@ export function isValidModelId(modelId: string): boolean {
   }
   return !isNaN(Number(modelId));
 }
+
+/**
+ * Extracts the modelId from the modelName.
+ * @param resourceName The full resource name of the model to extract the ID from.
+ * @return {string} The modelId portion of the model name.
+ */
+function extractModelId(resourceName: string): string {
+  return resourceName?.split("/").pop() || "";
+}
+
+/**
+ * Extract a status string from a model based on its state property.
+ * @param model The model to extract the status string from
+ * @return {string} The status of the model.
+ */
+function extractModelStatus(model: FirebaseModel): string {
+  if (model.state?.validationError) {
+    return "Invalid";
+  }
+  if (model.state?.published) {
+    return "Published";
+  }
+  return "Ready to publish";
+}
+
+/**
+ * Creates the display table for a model. (Used in GetModel.)
+ * @param model The model to create the display Table for.
+ * @return {Table} The display table.
+ */
+export function getTableForModel(model: FirebaseModel): Table {
+  const table = new Table(verticalTableFormat);
+  table.push({ modelId: extractModelId(model.name) }, { displayName: model.displayName });
+  if (model.tags) {
+    table.push({ tags: model.tags.join(", ") });
+  }
+  table.push({ status: extractModelStatus(model) });
+  table.push({ locked: (model.activeOperations !== undefined).toString() });
```

Comment:
it looks like this is an array? This would return a boolean that is being turned into a string. Is that intended?

Consider using `Array.isArray(mode.activeOperations)` - I think that will cleanly give you the boolean (which... I'm not sure if you need to cast to a string before doing that? maybe it's required by the types - and that's fine - but `!== undefined` is a little unconventional)

Reply:
I pulled it out into a helper function. We should actually be checking not just for array, but for non-empty array. Right now our server only gives either nothing (undefined) or an array with items, but that could change.